examples/oci/oci_gpu-sky.yaml

```diff
@@ -1,29 +1,303 @@
-name: cpu-task
+name: gpu-task1
 
 resources:
   # Optional; if left out, automatically pick the cheapest cloud.
   cloud: oci
 
+  accelerators: A10:1  # 1x NVIDIA A10 GPU
+
   region: ap-seoul-1
   
-  zone: bxtG:AP-SEOUL-1-AD-1
+  #zone: AP-SEOUL-1-AD-1
   
-  instance_type: VM.GPU.A10.1
+  #instance_type: VM.GPU.A10.1
 
-  image_id: skypilot:oci-ubuntu-NVIDIA-VMI-20_04
+  #image_id: skypilot:gpu-ubuntu-2004
```

Comment:
Maybe we can just remove these lines?

Reply:
Well i think it would be useful for user to have a reference value for these properties, though they are not mandatory.